addons/gut/gui/gut_config_gui.gd

```diff
@@ -272,8 +272,8 @@ func get_config_issues():
 	if(!has_directory):
 		to_return.append('You do not have any directories set.')
 
-	if(_cfg_ctrls['prefix'].text == ''):
-		to_return.append("You must set a Script prefix or GUT won't find any scripts")
+	if not ('.gd' in _cfg_ctrls['suffix'].text):
```

Comment:
Change this to `if(!_cfg_ctrls['suffix'].text.ends_with('.gd'))` to ensure it is just checking for the extension.  I don't really mind the `not` out front but it's more consistent with the rest of the code base to use `!`.

Reply:
Good call, Done.